### ISSUE TITLE
Figure mode

### ISSUE THREAD

**Issue by colinmegill** [chanzuckerberg/cellxgene#800]
We might allow users to customize figures, layering on extra information, when they want to screenshot and/or formally export. This functionality could be centralized in a menu. Issues that might fall into this bucket:

- [ ] #595 centroids
- [ ] #136 axis labels 
- [ ] #446 custom color palettes 
- [ ] #960 perfect circles

**Comment by cakirb** [comment]
By customization of the figures, did you also mean changing colors of the clusters? Because I have seen some people wish cellxgene had that kind of feature.

**Comment by colinmegill** [comment]
Yes, thanks you @cakirb, and we can continue to centralize issues here. I will keep the original issue edited/updated.

**Comment by signechambers1** [comment]
Closing since all linked sub-issues are closed
